src/base/CollectionBuilder.i.hh

```diff
@@ -53,14 +53,14 @@ auto PieBuilder<T, M, I>::insert_back(std::initializer_list<T> init)
  * Reserve space for the given number of elements.
  */
 template<class T, MemSpace M, class I>
-auto PieBuilder<T, M, I>::push_back(T el) -> PieIndexT
+auto CollectionBuilder<T, M, I>::push_back(T el) -> ItemIdT
 {
-    CELER_EXPECT(this->storage().size() + 1 <= this->max_pie_size());
+    CELER_EXPECT(this->storage().size() + 1 <= this->max_sizee());
```

Comment:
```suggestion
    CELER_EXPECT(this->storage().size() + 1 <= this->max_size());
```

Reply:
Ugh, sorry about these typos. My keyboard has been "sticky" lately -- lots of double presses. Could also be because I tend to have  a cat resting on my arm while I'm typing...